src/components/InputWithLabel.js

```diff
@@ -31,5 +32,8 @@ function InputWithLabel({ handleTitleChange, todoTitle, children, isFocused }) {
     </>
   );
 }
-
+InputWithLabel.propTypes = {
```

Comment:
Might be nice to have prop types for all the props - I don't see isFocused and children in the InputWithLabel.propTypes object